tests/fixtures/bucketfs_fixture.py

```diff
@@ -7,11 +9,47 @@
 
 
 @pytest.fixture(scope="session")
-def bucketfs_location(itde: TestConfig) -> bfs.path.PathLike:
-    return create_bucketfs_location(
-        path_in_bucket=bucketfs_params.path_in_bucket,
-        bucketfs_name=bucketfs_params.name,
-        bucketfs_url=itde.bucketfs.url,
-        bucketfs_user=itde.bucketfs.username,
-        bucketfs_password=itde.bucketfs.password,
-        bucket=bucketfs_params.bucket)
+def bucketfs_location_onprem(backend,
+                             itde: TestConfig) -> bfs.path.PathLike | None:
```

Comment:
Oops!!! I can see a much bigger problem. The SaaS counterpart is missing. It seems that the model is always uploaded into the DockerDB, even when running the SaaS tests. Not quite sure how it can work. It is actually amazing that it does.

Reply:
See comments below